apps/src/blockly/addons/cdoFieldLabel.ts

```diff
@@ -8,15 +8,27 @@ import * as GoogleBlockly from 'blockly/core';
 // Artist, and Play Lab. We can potentially add additional handling of
 // this argument to this class in the future should we need it.
 export default class CdoFieldLabel extends GoogleBlockly.FieldLabel {
+  // An override for legacy labs like Jigsaw where the rendered block size is constant.
+  fixedSize: {width: number; height: number} | undefined;
+
   constructor(
     value?: string | typeof GoogleBlockly.Field.SKIP_SETUP,
-    styleConfig?: string | object,
```

Comment:
Can you remind me why it was okay to drop these?

Reply:
CDO Blockly expects the value and custom size options.

Throughout our codebase, the second parameter has always been used to pass the size options, so it made sense to rename it. We don't have any instance where we provided a third `config` argument.

Google Blockly expects a value and (optionally) a class name and config. We don't provide class names.

There is additional context about how we got to this state here: https://github.com/code-dot-org/code-dot-org/pull/57778

The "Label Fields" section in particular details a step to make the custom class potentially support the mainline `textClass` argument too, by overloading the second parameter. This feels messy to me now, and unnecessary since it's not a thing we've ever tried to do.